sessions/class-01/exercise/start.js

```diff
@@ -45,7 +62,9 @@ export function getDiscountedValue(price, percentage) {}
  * @param {any} item The item to check against.
  * @returns {boolean} True if the item is falsy.
  */
-export function isFalsy(item) {}
+export function isFalsy(item) {
+    return Boolean(item) ? false : true;
```

Comment:
This works, as well

```suggestion
    return !item;
```

Because `!` will cast any value to boolean anyway.